bots/silverback_yield.py

```diff
@@ -3,85 +3,20 @@
 from ape import Contract, chain, networks
 from silverback import SilverbackApp
 
-from sqlmodel import SQLModel, Field, create_engine, Session
-
-sqlite_url = os.environ.get("DATABASE_URL")
-
-app = SilverbackApp()
+bot = SilverbackApp()
 
 with networks.ethereum.mainnet.use_provider("alchemy"):  # or "infura"
     vault = Contract(os.environ.get("ERC4626_VAULT_ADDRESS"))
```

Comment:
1. `SilverbackApp()` already connects to netwerk
2. don't use `.get` so it fails if the env var isn't available
```suggestion
vault = Contract(os.environ["ERC4626_VAULT_ADDRESS"])
```